수박수박수박수.js

```diff
@@ -0,0 +1,22 @@
+// 길이가 n 
+//문자열을 리턴하는 함수
+//n이 4면 수박수박 3이면 수박수
+function solution(n) {
+    //let 과 var의 차이점 (let은 중복 선언이 불가능 하고 var은 중복 선언이 가능하다. )
+    //결과값은 한번만 사용해도 괜찮으니까 let을 사용 한다. 
+        let answer = ""
+        
+    //for 문을 이용해서 문제 풀이 하는 방법
+        for (let i = 1; i<=n; i++ ) {// 선언과 동시에 값을 초기화 해준다.  
```

Comment:
제한조건이 있는게 아니라면 반복문에서 i는 0부터 시작해주세요!
추가로 이 문장도 컨벤션이요~
```suggestion
        for (let i = 0; i < n; i++ ) {
```